### PR TITLE
Fix power selector not showing up in user info when state_default undefined

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/element-web/issues/21669

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix power selector not showing up in user info when state_default undefined ([\#8297](https://github.com/matrix-org/matrix-react-sdk/pull/8297)). Fixes vector-im/element-web#21669.

Preview: https://pr8297--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/components/views/right_panel/UserInfo.tsx [hunk 1]
```diff
@@ -934,17 +929,14 @@ function useRoomPermissions(cli: MatrixClient, room: Room, user: RoomMember): IR
 
         let modifyLevelMax = -1;
         if (canAffectUser) {
-            const editPowerLevel = (
-                (powerLevels.events ? powerLevels.events["m.room.power_levels"] : null) ||
-                powerLevels.state_default
-            );
-            if (me.powerLevel >= editPowerLevel && (isMe || me.powerLevel > them.powerLevel)) {
```
Comment (turt2live) [inline src/components/views/right_panel/UserInfo.tsx:941]: I think we still need the `isMe || me > them` check to ensure the UI doesn't enable itself when looking at a fellow Admin?

Comment (t3chguy) [reply src/components/views/right_panel/UserInfo.tsx:941]: The check is done as part of `canAffectUser` which we're inside an if for

Comment (turt2live) [reply src/components/views/right_panel/UserInfo.tsx:941]: so it is

**Review by turt2live — APPROVED** [review]
apologies for the slow turnaround - admin work :(

**Pull request merged**